src/main/java/org/openmuc/jmbus/DataRecord.java

```diff
@@ -330,7 +330,7 @@ else if ((buffer[i + 1] & 0x80) == 0x80) {
                 calendar.set(year, mon - 1, day, hour, min, 0);
 
                 if (dst == 1) {
-                    calendar.set(Calendar.DST_OFFSET, 60000);
+                    calendar.set(Calendar.DST_OFFSET, 60000000);
```

Comment:
Wenn das die Angabe in ms ist, dann sollte hier 3600000 stehen:
1h = 60min * 60s * 1000ms = 3.600.000ms

Reply:
Hab den komplett entfernt.